openassessment/workflow/models.py

```diff
@@ -180,21 +180,6 @@ def score(self):
         """
         return sub_api.get_latest_score_for_submission(self.submission_uuid)
 
-    @property
-    def override_score(self):
-        """
-        Latest override score.
-        
-        Note an override score has no submission associated with it.
```

Comment:
Since you are merging into master, will there be an interim period between when this goes live and when the new implementation, via merge branch, is deployed.